tests/pds/dataservice_test.go

```diff
@@ -396,6 +396,111 @@ var _ = Describe("{ScaleUPDataServices}", func() {
+					}
+					log.InfoD("Non PDS MySQL App with name : %s is created", podName)
+				} else {
+					dash.VerifyFatal(result, true, "Failure in creating PVC on Target Cluster. Exiting the Test case with failure")
+				}
+			} else {
+				dash.VerifyFatal(result, true, "Failure in creating Persistent Volume on target cluster. Exiting the Test case with failure")
+			}
+		})
+		Step("Add PDS Label to Non-PDS Namespace running a DB Service already", func() {
+			nsLables := map[string]string{
+				pdsNamespaceLabel: "true",
+			}
+			testns, err := pdslib.UpdatePDSNamespce(ns, nsLables)
+			log.FailOnError(err, "Error while updating pds namespace")
+			log.InfoD("PDS Namespace Updated with PDS Label %v", testns)
+		})
+		Step("Deploy, Validate and Delete Data Services", func() {
+			for _, ds := range params.DataServiceToTest {
+				if ds.Name == postgresql {
+					log.InfoD("Deploying DataService %v ", ds.Name)
+					isDeploymentsDeleted = false
+					dataServiceDefaultResourceTemplateID, err = pdslib.GetResourceTemplate(tenantID, ds.Name)
+					log.FailOnError(err, "Error while getting resource template")
+					log.InfoD("dataServiceDefaultResourceTemplateID %v ", dataServiceDefaultResourceTemplateID)
+
+					dataServiceDefaultAppConfigID, err = pdslib.GetAppConfTemplate(tenantID, ds.Name)
+					log.FailOnError(err, "Error while getting app configuration template")
+					dash.VerifyFatal(dataServiceDefaultAppConfigID != "", true, "Validating dataServiceDefaultAppConfigID")
+					log.InfoD(" dataServiceDefaultAppConfigID %v ", dataServiceDefaultAppConfigID)
+					namespaceID, err := pdslib.GetnameSpaceID(ns, deploymentTargetID)
+					log.FailOnError(err, "error while getting namespaceid")
+					deployment, _, dataServiceVersionBuildMap, err = pdslib.DeployDataServices(ds.Name, projectID,
+						deploymentTargetID,
+						dnsZone,
+						deploymentName,
+						namespaceID,
+						dataServiceDefaultAppConfigID,
+						int32(ds.Replicas),
+						serviceType,
+						dataServiceDefaultResourceTemplateID,
+						storageTemplateID,
+						ds.Version,
+						ds.Image,
+						ns,
+					)
+					log.FailOnError(err, "Error while deploying data services")
+
+					Step("Validate Storage Configurations", func() {
+						resourceTemp, storageOp, config, err := pdslib.ValidateDataServiceVolumes(deployment, ds.Name, dataServiceDefaultResourceTemplateID, storageTemplateID, ns)
+						log.FailOnError(err, "error on ValidateDataServiceVolumes method")
+						ValidateDeployments(resourceTemp, storageOp, config, ds.Replicas, dataServiceVersionBuildMap)
+					})
+
+					Step("Delete Deployments", func() {
+						log.InfoD("Deleting DataService %v ", ds.Name)
+						resp, err := pdslib.DeleteDeployment(deployment.GetId())
+						log.FailOnError(err, "Error while deleting data services")
+						dash.VerifyFatal(resp.StatusCode, http.StatusAccepted, "validating the status response")
+						isDeploymentsDeleted = true
+					})
```

Comment:
where the variable isDeploymentsDeleted used?

Reply:
Done